### PR TITLE
[AAE-286] first implementation to secure end points for projects and models

### PR DESCRIPTION
**Pull request by CTI777**
First attempts to secure end points so that only creator can see projects and models

### CODE REVIEW

**Comment by magemello** [comment]
I think would be a much cleaner solution implement filters....example:

```
@WebFilter(urlPatterns = "/v1/projects/*")
public class ProjectPermissionFilter implements Filter {

    @Autowired
    private ProjectService projectService;

    @Override
    public void doFilter(ServletRequest servletRequest,
                         ServletResponse servletResponse,
                         FilterChain filterChain) throws IOException, ServletException {

        boolean permission = projectService.hasPersission(servletRequest);
        if (permission) {
            filterChain.doFilter(servletRequest,
                                 servletResponse);
        }else{
          returnInterruptingResponse(servletResponse,
                                          HttpStatus.FORBIDDEN);
                return;
       }
    }

    private void returnInterruptingResponse(ServletResponse servletResponse,
                                           HttpStatus forbidden) {
        HttpServletResponse resp = (HttpServletResponse) servletResponse;
        resp.reset();
        resp.setStatus(forbidden.value());
    }

    @Override
    public void destroy() {

    }
}

```

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/controller/ProjectController.java [hunk 1]
```diff
@@ -124,7 +131,41 @@ public void exportProject(
     }
 
     public Project findProjectById(String projectId) {
-        return projectService.findProjectById(projectId)
-                .orElseThrow(() -> new ResourceNotFoundException("Project not found: " + projectId));
+        
+        return findProjectWithCheckRights(projectId);
+    }
+    
+    private Project findProjectWithCheckRights(String projectId) {
```
Comment (magemello) [inline activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/controller/ProjectController.java:138]: please implement filters as from my comment

**Review by magemello — CHANGES_REQUESTED** [review]

File: activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/controller/ProjectController.java (hunk 1 above)
Comment (magemello) [reply activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/controller/ProjectController.java:138]: can you add also test to the PR?

**Comment by CTI777** [comment]
Test changes applied here are not valid, the whole new logic will be implemented in the modeling-service

**Pull request closed without merging**